pkg/provider/data_source_cluster.go

```diff
@@ -288,9 +290,32 @@ func (c *clusterDataSource) Schema(ctx context.Context, req datasource.SchemaReq
 				MarkdownDescription: "Earliest backup recover time.",
 				Computed:            true,
 			},
+			"maintenance_window": schema.SingleNestedAttribute{
+				Computed:            true,
+				MarkdownDescription: "Custom maintenance window.",
+				Attributes: map[string]schema.Attribute{
+					"is_enabled": schema.BoolAttribute{
+						MarkdownDescription: "Is maintenance window enabled.",
+						Computed:            true,
+					},
+					"start_day": schema.Float64Attribute{
+						MarkdownDescription: "Start day.",
```

Comment:
```suggestion
						MarkdownDescription: "The day of week, 0 represents Sunday, 1 is Monday, and so on.",
```
I directly copied from the API description [here](https://portal.biganimal.com/api/docs/#tag/Clusters/operation/createCluster).

If you incorporate this change, please also update in the other recurrences of start day of CMW in biganimal_cluster and biganimal_pgd resource/data-source.